cdap-app-fabric/src/main/java/io/cdap/cdap/app/guice/AppFabricServiceRuntimeModule.java

```diff
@@ -270,6 +273,7 @@ private AppFabricServiceModule(Class<? extends HttpHandler>... handlerClasses) {
     protected void configure() {
       bind(PipelineFactory.class).to(SynchronousPipelineFactory.class);
 
+      bind(PluginFinder.class).to(LocalPluginFinder.class);
```

Comment:
why do we need the change here and remove the ones in `InMemoryProgramRunnerModule`?

Reply:
After the change, LocalApplicationManager used by AppFabric and PreviewApplicationManager used by Preview requires this binding. But Preview uses AppFabricServiceRuntime module, so have to move the binding to AppFabricServiceRuntimeModule to satisfy the need by both appfab and preview.